package.json

```diff
@@ -32,6 +32,7 @@
     "@sentry/react": "^8.35.0",
     "@sentry/vite-plugin": "^2.22.6",
     "@tailwindcss/typography": "^0.5.13",
+    "@tanstack-dev/components": "^1.4.0",
```

Comment:
I don't think it would be a good idea to install a 3rd party dependency like this. This component should just live in this codebase.